packages/cluster/src/common.js

```diff
@@ -1,41 +1,48 @@
-const { reporter } = require('@dhis2/cli-helpers-engine')
+const path = require('path')
+const fs = require('fs')
+
+const { Cache, reporter } = require('@dhis2/cli-helpers-engine')
 
 const defaults = require('./defaults')
 
-const dockerComposeCacheName = 'd2-cluster-docker-compose-v2'
+const clusterDir = 'cluster'
```

Comment:
This could be inferred from `argv` since we have the command name. It does make it less obvious so went with readability++ for now. If we pull these out to "common for all of d2" it would make more sense to infer it.